next.config.js

```diff
@@ -1,9 +1,7 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   experimental: {
-    logging: {
-      level: 'verbose',
-    },
+    logging: 'verbose',
```

Comment:
@sherakama I updated this to string because I got a warning. I'm guessing you had more options before having to downgrade to 13.4.9? 😄 

![Screenshot 2023-10-18 at 12 56 16 PM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/ce2a0b11-17d2-4057-aa79-da0e8a70b62c)
![Screenshot 2023-10-18 at 12 56 29 PM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/f237081c-dc0c-4b49-8463-2e6535ff96b7)